http-scala-fx/src/main/scala/fx/MediaType.scala

```diff
@@ -0,0 +1,11 @@
+package fx
+
+opaque type MediaType = String
+
+extension (m: MediaType)
+  def value: String =
+    m
+
+object MediaType:
+  def apply(s: String): MediaType =
+    s
```

Comment:
We're getting this warning about a duplicated top-level definition of the `value` function:
```
[warn] Toplevel definition value is defined in
[warn]   ~/http-scala-fx/target/scala-3.1.2/classes/fx/MediaType$package.class
[warn] and also in
[warn]   ~/scala-fx/target/scala-3.1.2/classes/fx/Nullable$package.class
[warn] Keeping only the definition in ~/http-scala-fx/target/scala-3.1.2/classes/fx/MediaType$package.class
```
So we could put this extension method for `MediaType` within its companion object to eliminate this warning message.
```suggestion
    s

  extension (m: MediaType)
    def value: String =
      m
```

Reply:
Resovled in [b956f94](https://github.com/47deg/TBD/pull/37/commits/b956f94aaf664dc98941de86fb46b52ef702b3ae)